src/components/profile/VolunteeringHistory.jsx

```diff
@@ -7,8 +8,7 @@ import EditHours from '../volunteer-profile-history/EditHours';
 
 const { Title } = Typography;
 
-function VolunteeringHistory() {
-  const [userId, setUserId] = useState(2);
+function VolunteeringHistory({ userId }) {
```

Comment:
Let's make this a full arrow function since you're changing the signature on this line anyways